docs/architecture/adr/0025-ts-deprecate-enums.md

```diff
@@ -1,7 +1,8 @@
 ---
 adr: "0025"
-status: Proposed
+status: Accepted
 date: 2025-05-30
+accepted: 2025-06-24
```

Comment:
ℹ️ I'd like to attempt to get past dates and fill these out everywhere if we go forward with this.